example/fly_mission/CMakeLists.txt

```diff
@@ -6,34 +6,12 @@ if(NOT MSVC)
     add_definitions("-std=c++11 -Wall -Wextra -Werror")
 else()
     add_definitions("-std=c++11 -WX -W2")
-    set(platform_libs "Ws2_32.lib")
 endif()
 
-# Add DEBUG define for Debug target
-set(CMAKE_CXX_FLAGS_DEBUG "-DDEBUG")
```

Comment:
Release is built by default. Is this removed because we're happy for release build to be the default?

Reply:
This is not actually about the default. All this does is it adds a `#define DEBUG` in case it is a debug build. This is not actually used or needed in the examples, so I removed it.

Reply:
Thanks for the clarification.